src/components/tableRow.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+
+function TableRow(props) {
+  const { key, missionName, missionDesc } = props;
+  return (
+    <tr key={key}>
```

Comment:
The key which is passed in ```Missions.js``` is not a prop, so you can not access it here, and ```<tr>``` doesn't need it. Can you please remove the key on this file, just pass it as you did in the ```Missions.js``` file, that's all you needed,